.gitmodules

```
@@ -1,4 +1,4 @@
 [submodule "dolly"]
 	path = dolly
-	url = https://github.com/BlackPhlox/dolly
-    branch = bevy_dolly
\ No newline at end of file
+	url = https://github.com/rparrett/dolly
+    branch = bevy-0.14
```

I've bumped the bevy_* dependencies on my fork, so for convenience: 
```suggestion
	url = https://github.com/BlackPhlox/dolly
    branch = bevy_0.14
```